addons/core/translations/form/en-us.yaml

```diff
@@ -115,3 +115,12 @@ value:
 callback_url:
   label: Callback URL
   help: The URL to which the third-party provider sends the authentication response.  Paste this into the provider configuration.
+vault_path:
+  label: Vault Path
+  help: The path in Vault to request credentials from.
+http_method:
+  label: HTTP Method
+  help: The HTTP method the library uses when requesting credentials from Vault.
```

Comment:
The field name `http_method` is too generic to mention a specific resource type or cred store type, or particular validation constraints.  I'm not sure how to handle this situation.  Thoughts?